eva-lib/src/test/java/uk/ac/ebi/eva/lib/repository/VariantEntityRepositoryTest.java

```diff
@@ -420,12 +421,12 @@ public void testFindByRegionsAndComplexFiltersExcludeAttributes() {
     public void testFindDistinctChromosomesByStudyId() {
         List<String> chromosomeList = variantEntityRepository.findDistinctChromosomes();
```

Comment:
Given these are distinct, it would more sense so return a Set instead of a List.